[BUG] [CPP] [pistache-server] Generated code fails to build with  x86_64-linux-gnu-g++-13

#### Bug Report Checklist

- [X] Have you provided a full/minimal spec to reproduce the issue?
- [X] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [X] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [X] Have you searched for related issues/PRs?
- [x] What's the actual output vs expected output?

<!--
Please follow the issue template below for bug reports.
Also please indicate in the issue title which language/library is concerned. Eg:  [BUG][JAVA] Bug generating foo with bar 
-->

##### Description

Generated sources for cpp-pistache-server, fails to compile with x86_64-linux-gnu-g++-13 . The particular issue is

Actual output:
```
[ 95%] Building CXX object CMakeFiles/api-server.dir/main-api-server.cpp.o                                                                                                                                                                
/tmp/foo/cpp/model/Helpers.cpp:28:27: error: ‘uint16_t’ does not name a type                                                                                                                                                              
   28 |     bool isLeapYear(const uint16_t year) {                                                                                                                                                                                        
      |                           ^~~~~~~~                                                                                                                                                                                                
/tmp/foo/cpp/model/Helpers.cpp:14:1: note: ‘uint16_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?                                                                                                           
   13 | #include <regex>                                                                                                                                                                                                                  
  +++ |+#include <cstdint>                                                                                                                                                                                                                
   14 | 

```

Expected output:

```
...
...
[ 82%] Building CXX object CMakeFiles/api-server.dir/model/Helpers.cpp.o
[ 82%] Building CXX object CMakeFiles/api-server.dir/api/SecurityApi.cpp.o
[ 86%] Linking CXX executable api-server
[100%] Built target api-server
```

##### openapi-generator version

master @ 29c3b020e620330669bac9950863b09bc2e61764

##### OpenAPI declaration file content or url

```yaml
openapi: 3.0.2
servers:
  - url: http://localhost/v3
    description: Localhost. Might use live data.

info:
  description: |-
    This is a demonstration API
  version: 1.0.1
  title: Fubar API
  termsOfService: 'http://foo.bar.com/tos'
  contact:
    email: bartender@foo.bar.com

tags:
  - name: Foos
    description: Anything related to Foos

paths:

  /foo:

    get:
      summary: Returns a foo
      operationId: returnFoo
      tags: [ Foos ]

      responses:
        '200':
          description: A Foo fresh from the Bar.
          content:
            text/plain:
              schema:
                type: string
                example: "Foo"
```

##### Generation Details



##### Steps to reproduce

```
java -jar modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g cpp-pistache-server  -i api.yaml  -o cpp
cd cpp
cmake .
make -j
```

##### Related issues/PRs

<!-- has a similar issue/PR been reported/opened before? Please do a search in https://github.com/openapitools/openapi-generator/issues?utf8=%E2%9C%93&q=is%3Aissue%20 -->

##### Suggest a fix

I suggest adding 
```c++
#include <cstdint>
```

to the relevant Mustache template.

--- a/modules/openapi-generator/src/main/resources/cpp-pistache-server/helpers-header.mustache
+++ b/modules/openapi-generator/src/main/resources/cpp-pistache-server/helpers-header.mustache
@@ -10,11 +10,11 @@
 
 #include <cstdint>
 #include <ctime>
-#include <string>
-#include <sstream>
-#include <vector>
 #include <map>
 #include <set>
+#include <sstream>
+#include <string>
+#include <vector>
 
 namespace {{helpersNamespace}}
 {

--- a/modules/openapi-generator/src/main/resources/cpp-pistache-server/helpers-header.mustache
+++ b/modules/openapi-generator/src/main/resources/cpp-pistache-server/helpers-header.mustache
@@ -8,6 +8,7 @@
 #ifndef {{prefix}}Helpers_H_
 #define {{prefix}}Helpers_H_
 
+#include <cstdint>
 #include <ctime>
 #include <string>
 #include <sstream>

--- a/samples/server/petstore/cpp-pistache-everything/model/Helpers.h
+++ b/samples/server/petstore/cpp-pistache-everything/model/Helpers.h
@@ -18,12 +18,13 @@
 #ifndef Helpers_H_
 #define Helpers_H_
 
+#include <cstdint>
 #include <ctime>
-#include <string>
-#include <sstream>
-#include <vector>
 #include <map>
 #include <set>
+#include <sstream>
+#include <string>
+#include <vector>
 
 namespace org::openapitools::server::helpers
 {

--- a/samples/server/petstore/cpp-pistache-nested-schema-refs/model/Helpers.h
+++ b/samples/server/petstore/cpp-pistache-nested-schema-refs/model/Helpers.h
@@ -18,12 +18,13 @@
 #ifndef Helpers_H_
 #define Helpers_H_
 
+#include <cstdint>
 #include <ctime>
-#include <string>
-#include <sstream>
-#include <vector>
 #include <map>
 #include <set>
+#include <sstream>
+#include <string>
+#include <vector>
 
 namespace org::openapitools::server::helpers
 {

--- a/samples/server/petstore/cpp-pistache/model/Helpers.h
+++ b/samples/server/petstore/cpp-pistache/model/Helpers.h
@@ -18,12 +18,13 @@
 #ifndef Helpers_H_
 #define Helpers_H_
 
+#include <cstdint>
 #include <ctime>
-#include <string>
-#include <sstream>
-#include <vector>
 #include <map>
 #include <set>
+#include <sstream>
+#include <string>
+#include <vector>
 
 namespace org::openapitools::server::helpers
 {

